specs/core/0_beacon-chain.md

```diff
@@ -702,7 +700,8 @@ def get_previous_epoch(state: BeaconState) -> Epoch:
     """`
     Return the previous epoch of the given ``state``.
```

Comment:
It would be nice to update the docstring (again 😅)
```suggestion
    Return the previous epoch of the given ``state``.
    Return the current epoch if it's genesis epoch.
```